### PR TITLE
media-sound/milkytracker: Add 1.02.00 (security)

### PR DESCRIPTION
**Pull request by ajakk**
Bug: https://bugs.gentoo.org/711280
Closes: https://bugs.gentoo.org/711564
Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: John Helmert III <jchelmert3@posteo.net>

### CODE REVIEW

File: media-sound/milkytracker/milkytracker-1.02.00.ebuild [hunk 1]
```diff
@@ -23,8 +28,10 @@ RDEPEND="
 DEPEND="${RDEPEND}"
 
 PATCHES=(
```
Comment (thesamesam) [inline media-sound/milkytracker/milkytracker-1.02.00.ebuild:30]: Put a header line in all of the patches with where they've been found.

File: media-sound/milkytracker/milkytracker-1.02.00.ebuild [hunk 2]
```diff
@@ -23,8 +28,10 @@ RDEPEND="
...
-	"${FILESDIR}"/${P}-docdir.patch
-	"${FILESDIR}"/${P}-cmake.patch
+	"${DISTDIR}/${P}-cmake.patch"
+	"${FILESDIR}/${P}-useafterfree-fix.patch"
```
Comment (thesamesam) [inline media-sound/milkytracker/milkytracker-1.02.00.ebuild:32]: What does this fix? No CVE?

File: media-sound/milkytracker/milkytracker-1.02.00.ebuild [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
 inherit cmake desktop
 
+# This commit is needed so the milkytrace binary is linked properly
+# Bug 711562
+COMMIT="2b145b074581ddf3b4ad78a402cdf5fab500b125"
+
 DESCRIPTION="FastTracker 2 inspired music tracker"
 HOMEPAGE="https://milkytracker.titandemo.org/"
-SRC_URI="https://github.com/milkytracker/MilkyTracker/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/milkytracker/MilkyTracker/archive/v${PV}.tar.gz -> ${P}.tar.gz
```
Comment (thesamesam) [inline media-sound/milkytracker/milkytracker-1.02.00.ebuild:14]: I guess the cmake patch is too big to include here?

Comment (ajakk) [reply media-sound/milkytracker/milkytracker-1.02.00.ebuild:14]: Yes, ~40kb, `repoman` complained about it.

File: media-sound/milkytracker/milkytracker-1.02.00.ebuild (hunk 2 above)
Comment (ajakk) [reply media-sound/milkytracker/milkytracker-1.02.00.ebuild:32]: Whoops, filed this PR before the CVE was assigned and forgot to adjust. Will correct it.

**Pull request closed without merging**